src/Lean/Elab/Tactic/AsAuxLemma.lean

```diff
@@ -0,0 +1,26 @@
+/-
+Copyright (c) 2022 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Reichert
```

Comment:
Presumably this should assign copy-right to Lean FRO, LLC in 2024 and list Joachim Breitner as the author.